package.json

```diff
@@ -6,6 +6,7 @@
     "files": [
         "dist"
     ],
+    "types": "dist/index.d.ts",
```

Comment:
There is probably something to do with your bundler so it forwards the type file into the published npm package.